cli/src/generate/node_types.rs

```diff
@@ -328,10 +328,9 @@ pub(crate) fn get_variable_info(
     for supertype_symbol in &syntax_grammar.supertype_symbols {
         if result[supertype_symbol.index].has_multi_step_production {
             let variable = &syntax_grammar.variables[supertype_symbol.index];
-            return Err(Error::grammar(&format!(
-                "Supertype symbols must always have a single visible child, but `{}` can have multiple",
+            return Err(anyhow!( "Grammar error: Supertype symbols must always have a single visible child, but `{}` can have multiple",
```

Comment:
Is there a stray space in there before the string?

Reply:
Yeah I guess the long string literal confused `rustfmt`?  I added in a `concat!` to make it happy